.github/workflows/e2e-connectivity.yaml

```diff
@@ -0,0 +1,222 @@
+name: NetworkPolicy tests
+
+on:
+  push:
+    branches: [ "main" ]
+
+  pull_request:
+    branches: [ "main", "release-*" ]
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./src/knative.dev/serving
+
+jobs:
+  e2e-connectivity:
+    name: e2e connectivity
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s-version:
+        - v1.22.0
```

Comment:
Let's pull in the latest patch

https://hub.docker.com/layers/kindest/node/v1.22.4/images/sha256-ca3587e6e545a96c07bf82e2c46503d9ef86fc704f44c17577fca7bcabf5f978?context=explore